mvlearn/cotraining/ctclassifier.py

```diff
@@ -113,6 +119,40 @@ class CTClassifier(BaseCoTrainEstimator):
         The starting random seed for fit() and class operations, passed to
         numpy.random.seed().
 
+    Examples
```

Comment:
This seems long. Off of the top of my head I don't recall seeing any sklearn like documentation this long but could be wrong. Tutorial vs this tradeoff. Up to you.

Reply:
Also, I assume that you compiled the docs and all of these look fine. There is a way for tests to run and compile these, I am not sure if we have that implemented in our travis though.